### PR TITLE
Garage: Add Pilot page

### PR DESCRIPTION
**Pull request by datadanne**
Adds a detail page for pilots.
image

Links from RigDetails Pilot module
image

Closes https://github.com/tablelandnetwork/rigs/issues/395

### CODE REVIEW

File: garage/src/routes.tsx [hunk 1]
```diff
@@ -25,5 +26,11 @@ export const routes = () => {
       requiresWalletConnection: false,
       element: <OwnerDetails />,
     },
+    {
+      key: "PILOT_DETAILS",
+      path: "/pilot/:collection/:id",
```
Comment (sanderpick) [inline garage/src/routes.tsx:31]: how about plural instead? ie, `"/pilots/:collection/:id"`

**Review by sanderpick — COMMENTED** [review]
left a small comment.

as a side note, I think it would look nice to add a little extra space below the top nav bar to separate it from the main content (like 2x what is there now), at least on larger screens. the pilot page kinda drives that home since they will be so short in height

File: garage/src/routes.tsx (hunk 1 above)
Comment (datadanne) [reply garage/src/routes.tsx:31]: Sure!

**Comment by datadanne** [comment]
> as a side note, I think it would look nice to add a little extra space below the top nav bar to separate it from the main content (like 2x what is there now), at least on larger screens. the pilot page kinda drives that home since they will be so short in height

Yeah that sounds like a good idea, I'll play with that!

**Comment by datadanne** [comment]
@sanderpick I doubled the padding between the topbar and content on all pages and it looks good imo! Pushed some new commits that changes the top padding + changes the route name + adds a missing link from the collector profile pilots module to the new pilot page

**Review by sanderpick — APPROVED** [review]
lovely!

**Comment by datadanne** [comment]
@sanderpick could you re-review this one? I had to rebase it on `main` to get the tests to pass

**Review by sanderpick — APPROVED** [review]
👍

**Pull request merged**